ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,287 @@
+SITE_HOST: 'enikshay.in'
+#CAS_SITE_HOST: 'cas.commcarehq.org'
+#ENIKSHAY_SITE_HOST: 'enikshay.commcarehq.org'
```

Comment:
Thanks for pointing that out, what about ENIKSHAY_SITE_HOST, it won't be used any longer, right? Sice the main site will be enikshay...